lib/make_call.rb

```diff
@@ -0,0 +1,19 @@
+require 'twilio-ruby'
+require_relative 'order'
+class Checkout
+
+  def initialize
+    account_sid = 'ACdb0c001d98d842c9465935ffde45fb6b'
+    auth_token = 'aa648095dec33b70637f15de4c6c9964'
```

Comment:
You should never commit auth tokens to a public repository.  You ought to change your token on Twilio for safety.